### PR TITLE
[Feat]: Videos list

### PR DESCRIPTION
**Pull request by Logan1x**
- Video Details page
- added favicon
- login feature( add to playlist, add to watch later, etc)
- sidebar

### CODE REVIEW

**Review by vinc3nati — APPROVED** [review]
Code looks modular, concise code.
Suggestion:
Add Screenshots or a video of the feature you've added so it's easier to assess

File: src/pages/home/home.css [hunk 1]
```diff
@@ -32,6 +32,21 @@ p {
...
+  border: none;
+}
+
+
+.home-title-btn:hover {
+  opacity: 0.85;
+}
+
 .section-category {
   /* padding: 0.5rem; */
```
Comment (surajmore98) [inline src/pages/home/home.css]: you can remove this comment

File: src/pages/videos/videos.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    height: 100%;
+    margin: 1rem;
+    padding: 0.5rem;
+    border: 1px solid;
+    border-radius: 0.2rem;
+}
+
+.videoListing-card-image{
+        object-fit: cover;
+        /* height: 100%; */
```
Comment (surajmore98) [inline src/pages/videos/videos.css]: remove this comment also.

File: src/pages/videos/videos.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        width: 20rem;
+}
+
+.videoListing-card:hover{
+    box-shadow: 1px 2px 5px 0px #BEC0D8;
```
Comment (surajmore98) [inline src/pages/videos/videos.css:34]: you can create variable for this color to maintain consistency.

**Review by surajmore98 — APPROVED** [review]
well structured and easy to understandable code. 
good to merge with above suggestion.

File: src/pages/videos/videos.css (hunk 3 above)
Comment (Logan1x) [reply src/pages/videos/videos.css:34]: This is being used only once. SO I think it's fine. I'll take care of it in the next iteration.

**Comment by Logan1x** [comment]
Thanks, Everyone, Merging now :)

**Pull request merged**